pmd-java/src/main/resources/category/java/bestpractices.xml

```diff
@@ -459,7 +460,7 @@ JUnit 4 skips the tearDown method and executes all methods annotated with @After
 <![CDATA[
 //CompilationUnit[not(ImportDeclaration/Name[starts-with(@Image, "org.testng")])]
 //ClassOrInterfaceBodyDeclaration[MethodDeclaration/MethodDeclarator[@Image='tearDown']]
-[count(Annotation//Name[@Image='After'])=0]
+[count(Annotation//Name[@Image='After' or @Image='AfterEach' or @Image='AfterAll'])=0]
```

Comment:
totally unrelated to this PR,  but when merging we should use `typeIs` here /cc @adangel

Reply:
There are many rules here, that don't use typeresolution yet.... I can give it a try - if we are lucky, the test cases already import the correct classes....